modin/pandas/test/test_series.py

```diff
@@ -2778,13 +2778,12 @@ def test_var(data, skipna, ddof):
         df_equals(modin_result, pandas_result)
 
 
-@pytest.mark.skipif(sys.platform == "win32", reason="Windows memory issue #960")
-@pytest.mark.parametrize("data", test_data_values, ids=test_data_keys)
-def test_view(data):
-    modin_series, pandas_series = create_test_series(data)
-
-    with pytest.warns(UserWarning):
-        modin_series.view(None)
+def test_view():
```

Comment:
It might be good to also test with `float64`.

Reply:
Of course, done.